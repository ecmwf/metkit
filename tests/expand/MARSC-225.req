retrieve,
class=ti,date=2023-12-30,expver=prod,grid=0.2/0.2,levtype=sfc,padding=0,param=167,step=0/to/360/by/6,time=00:00:00,type=pf,number=1,origin=ecmf,
target="reference.8nWO8K.data"