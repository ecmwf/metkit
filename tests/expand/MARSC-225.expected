RETRIEVE,
            CLASS      = TI,
            TYPE       = PF,
            STREAM     = ENFO,
            EXPVER     = prod,
            REPRES     = SH,
            LEVTYPE    = SFC,
            PARAM      = 167,
            DATE       = 20231230,
            TIME       = 0000,
            STEP       = 0/6/12/18/24/30/36/42/48/54/60/66/72/78/84/90/96/102/108/114/120/126/132/138/144/150/156/162/168/174/180/186/192/198/204/210/216/222/228/234/240/246/252/258/264/270/276/282/288/294/300/306/312/318/324/330/336/342/348/354/360,
            NUMBER     = 1,
            ORIGIN     = ECMF,
            TARGET     = reference.8nWO8K.data,
            RESOL      = AUTO,
            GRID       = .2/.2,
            PADDING    = 0,
            PROCESS    = LOCAL